test/generators/extra_pages_generator_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class ExtraPagesGeneratorTest < Rails::Generators::TestCase
+  tests MountainView::Generators::ExtraPagesGenerator
+
+  destination File.expand_path("../../tmp2", __FILE__)
```

Comment:
Nothing special to have a different place where run the tests. You think it has something . to do with the tests failing?